skfda/representation/basis.py

```diff
@@ -325,6 +325,70 @@ def _list_to_R(self, knots):
     def _to_R(self):
         raise NotImplementedError
 
+    def inner_matrix(self, other=None):
```

Comment:
Should it be public?

Reply:
I think different with this, I don't know how this method could help. I'll put it private.